Std/Linter/UnreachableTactic.lean

```diff
@@ -58,6 +58,8 @@ This should be called from an `initialize` block.
 def addIgnoreTacticKind (kind : SyntaxNodeKind) : IO Unit :=
   ignoreTacticKindsRef.modify (·.insert kind)
 
+initialize Std.Linter.UnreachableTactic.addIgnoreTacticKind `Lean.Parser.Tactic.discharger
```

Comment:
If it's in the same file, you may as well add it to the `HashSet` directly (line 46). Also always use double-backtick if possible.

Reply:
Oh of course, thanks :), done